### PR TITLE
ISPN-13490 HotRod 3.1 UpdateBloomFilterOperation is not documented in…

### PR DESCRIPTION
**Pull request by wburns**
… protocol guide

https://issues.redhat.com/browse/ISPN-13490

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/hotrod_protocol.adoc [hunk 1]
```diff
@@ -3171,6 +3171,25 @@ server to keep track of the near cache in a bloom filter. This allows the server
...
 
+Response (0x42):
+
+[cols="3,^2,10",options="header"]
+|==============================================================================
+| Field Name          | Size       | Value
+| Header | variable | Response header
+|==============================================================================
+
+.update bloom filter
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/hotrod_protocol.adoc:3182]:
```suggestion
.Update bloom filter
```

**Review by oraNod — CHANGES_REQUESTED** [review]
@wburns Pretty minor suggestion but looks good otherwise.

**Comment by wburns** [comment]
Updated to capitalize both names for bloom docs

**Review by oraNod — APPROVED** [review]
LGTM. Thanks @wburns

**Pull request merged**